example/fileupload/server/server.go

```diff
@@ -55,46 +46,52 @@ func getResolver() *fileupload.Stub {
 			return nil, err
 		}
 		return &model.File{
-			ID:      1,
-			Name:    req.File.Filename,
-			Content: string(content),
+			ID:          1,
+			Name:        req.File.Filename,
+			Content:     string(content),
+			ContentType: req.File.ContentType,
 		}, nil
 	}
 	resolver.MutationResolver.MultipleUpload = func(ctx context.Context, files []*graphql.Upload) ([]*model.File, error) {
 		if len(files) == 0 {
 			return nil, errors.New("empty list")
 		}
 		var resp []*model.File
-		for i := range files {
-			content, err := ioutil.ReadAll(files[i].File)
+		for i, file := range files {
+			content, err := ioutil.ReadAll(file.File)
 			if err != nil {
 				return []*model.File{}, err
 			}
 			resp = append(resp, &model.File{
-				ID:      i + 1,
-				Name:    files[i].Filename,
-				Content: string(content),
+				ID:          i + 1,
+				Name:        file.Filename,
+				Content:     string(content),
+				ContentType: file.ContentType,
 			})
 		}
 		return resp, nil
 	}
-	resolver.MutationResolver.MultipleUploadWithPayload = func(ctx context.Context, req []*model.UploadFile) ([]*model.File, error) {
-		if len(req) == 0 {
+	resolver.MutationResolver.MultipleUploadWithPayload = func(ctx context.Context, reqs []*model.UploadFile) ([]*model.File, error) {
+		if len(reqs) == 0 {
 			return nil, errors.New("empty list")
 		}
 		var resp []*model.File
-		for i := range req {
-			content, err := ioutil.ReadAll(req[i].File.File)
+		for i, req := range reqs {
+			content, err := ioutil.ReadAll(req.File.File)
 			if err != nil {
 				return []*model.File{}, err
 			}
 			resp = append(resp, &model.File{
-				ID:      i + 1,
-				Name:    req[i].File.Filename,
-				Content: string(content),
+				ID:          i + 1,
+				Name:        req.File.Filename,
+				Content:     string(content),
+				ContentType: req.File.ContentType,
```

Comment:
these changes seem unrelated?

Reply:
According to https://github.com/99designs/gqlgen/pull/1074 changes, I think it is related.